src/pages/create-or-edit-lesson/CreateOrEditLesson.tsx

```diff
@@ -60,23 +59,26 @@ const CreateOrEditLesson = () => {
   const navigate = useNavigate()
   const { id } = useParams()
 
-  const handleResponseError = (error?: ErrorResponse) => {
-    const errorKey = getErrorKey(error)
+  const handleResponseError = useCallback(
+    (error?: ErrorResponse | ResponseError) => {
```

Comment:
You can leave it here like that because there is only `ResponseError` used. Also it's never optional
```suggestion
    (error: ResponseError) => {
```